osutil/user/user_from_snap.go

```diff
@@ -0,0 +1,74 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+//go:build snap
```

Comment:
hm not sure about the build tag, maybe @pedronis has an opinion about it

I would try something more specific, eg `snapdusergo` ?

Reply:
I have renamed it.